### PR TITLE
Mybranch

### PR DESCRIPTION
**Pull request by RealDidier**
This is my last and best pull request. 
Thanks.

### CODE REVIEW

File: portfolio/src/main/java/DataServlet.java [hunk 1]
```diff
@@ -15,19 +15,40 @@
...
 import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+/** Servlet that returns HTML that contains the page view count. */
```
Comment (mohandasks) [inline portfolio/src/main/java/DataServlet.java]: Looks like the comment is not in sync with what this class is doing?

File: portfolio/src/main/java/DataServlet.java [hunk 2]
```diff
@@ -15,19 +15,40 @@
...
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
-    response.getWriter().println("Hello Didier!"); 
+   
+    //String randomSentence = random.get((int) (Math.random()*random.size()) ); 
+    response.setContentType("text/html");
+    response.getWriter().println(random);
```
Comment (mohandasks) [inline portfolio/src/main/java/DataServlet.java]: Is it intentional to print the entire list?

File: portfolio/src/main/java/DataServlet.java [hunk 3]
```diff
@@ -15,19 +15,40 @@
...
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+/** Servlet that returns HTML that contains the page view count. */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+    private List<String> random;
```
Comment (mohandasks) [inline portfolio/src/main/java/DataServlet.java]: Wouldn't the application container create an instance of Servlet for every request? So this list won't be persisted across requests, right?

Comment (RealDidier) [reply portfolio/src/main/java/DataServlet.java]: Modified that ago. Just did another commit.

File: portfolio/src/main/java/DataServlet.java (hunk 2 above)
Comment (RealDidier) [reply portfolio/src/main/java/DataServlet.java]: Yeah

File: portfolio/src/main/java/DataServlet.java (hunk 1 above)
Comment (RealDidier) [reply portfolio/src/main/java/DataServlet.java]: I deleted the original file. I  modified and used the file from the walkthrough.

**Comment by RealDidier** [comment]
Just did another commit.

**Pull request merged**